src/pager.c

```diff
@@ -652,6 +652,14 @@ lookat(int x, int y, char *buf, char *monbuf)
             else
                 Strcpy(buf, "doorway");
             break;
+        case S_vcdoor:
+        case S_hcdoor:
+            if (IS_DOOR(levl[x][y].typ) && next2u(x, y)
+                && door_is_locked(&levl[x][y]))
+                Strcpy(buf, "locked door");
+            else
+                Strcpy(buf, "closed door");
```

Comment:
Rather than hardcoding "closed door", this should probably use the default case of using `defsyms[symidx].explanation` (not that it should FALLTHRU to the default case, since S_stone is already doing that).

Reply:
Yeah, I spent some time vacillating about whether to do that or not but basically figured this matched what other cases like `S_stone` and `S_ndoor` do.  I can update it.